gradle.properties

```diff
@@ -1,5 +1,5 @@
 version=7.1.0
-iexecCommonVersion=6.0.0
+iexecCommonVersion=6.0.1
 iexecBlockchainAdapterVersion=7.1.1
 iexecResultVersion=7.1.0
 iexecSmsVersion=7.1.0
```

Comment:
Shouldn't some of those dependencies be in `-NEXT-SNAPSHOT` ?
I see at least `iexecCommonVersion` and `iexecSmsVersion`.

Reply:
I agree :)